R/sfi_plot_laqueur_2.R

```diff
@@ -15,163 +15,192 @@ sfi_plot_laqueur_2 <- function(){
-    geom_point(size = 3, 
-               alpha = 0.6) +
-    labs(x = 'Year',
-         y = 'Rate of parole grant',
-         title = 'Version 2') +
-    theme_sfi()
+
   
-  # version 3 
-  g3 <- ggplot(data, 
-               aes(x = year, 
-                   y = percent_of_conducted_hearings_resulting_in_a_grant)) +
-    geom_point(size = 3, 
-               alpha = 0.6) +
-    geom_line(size = 1, 
-              alpha = 0.6) +    
+  # version 2
+  g2 <- 
+    ggplot(data, 
+           aes(x = year, 
+               y = percent_of_conducted_hearings_resulting_in_a_grant)) +
+    geom_smooth(method = 'loess',
+                alpha = 0.7,
+                color = 'black',
+                fill = 'black',
+                linetype = 0) + 
+    geom_point(aes(size = percent_of_conducted_hearings_resulting_in_a_grant), 
+               color = 'black',
+               fill = 'white',
+               alpha = 0.8,
+               pch = 21) +
+    geom_text(aes(label = paste0(percent_of_conducted_hearings_resulting_in_a_grant, 
+                                 '%'),
+                  size = percent_of_conducted_hearings_resulting_in_a_grant/8),
+              nudge_x = 0) +
+    scale_size_area(name = '',
+                    max_size = 30) +
+    xlim(c(2007, 2015)) + ylim(c(0, 50)) +
+    theme(legend.position="none") +
     labs(x = 'Year',
-         y = 'Rate of parole grant',
-         title = 'Version 3') +
-    theme_sfi()
+         y = 'Hearings resulting in grant',
+         title = 'Version 2',
+         caption = paste0('Smoothed with a local regression', '\n', 
+                          'with bands representing standard errors')) +
+    theme_sfi(lp = 'none')
+
+  # version 3
+  g3 <- 
+    ggplot(data, 
+           aes(x = year, 
+               y = percent_of_conducted_hearings_resulting_in_a_grant)) +
+    geom_point(aes(size = percent_of_conducted_hearings_resulting_in_a_grant), 
+               color = 'grey',
+               fill = 'black',
+               stroke = 2,
+               alpha = 0.7,
+               pch = 21) +
+    geom_text(aes(label = paste0(percent_of_conducted_hearings_resulting_in_a_grant, 
+                                 '%'),
+              size = percent_of_conducted_hearings_resulting_in_a_grant/8),
+              color = 'white',
+              nudge_x = 0) +
+    scale_size_area(name = '',
+                    max_size = 30) +
+    xlim(c(2007, 2015)) + ylim(c(0, 50)) +
+    theme(legend.position="none") +
+    labs(x = 'Year',
+         y = 'Hearings resulting in grant',
+         title = 'Version 3',
+         caption = paste0('Smoothed with a local regression', '\n', 
+                         'with bands representing standard errors')) +
+    theme_sfi(lp = 'none', )
+  
   
   # version 4
-  g4 <- ggplot(data, 
-               aes(x = year, 
-                   y = percent_of_conducted_hearings_resulting_in_a_grant)) +
-    geom_smooth(method = 'lm',
-                size = 0.5, 
+  g4 <- 
```

Comment:
Same commnt as before - I have hesitations about using confidence bands when there are so few observations.

Reply:
They really liked those bands for some reason, without the mean line. I think it's overkill, but that's what they asked for.